src/OpenApi/gen/XmlCommentGenerator.Emitter.cs

```diff
@@ -416,6 +441,49 @@ public Task TransformAsync(OpenApiOperation operation, OpenApiOperationTransform
                     }
                 }
             }
+            foreach (var parameterDescription in context.Description.ParameterDescriptions)
+            {
+                var metadata = parameterDescription.ModelMetadata;
+                if (metadata.MetadataKind == ModelMetadataKind.Property
+                    && metadata.ContainerType is { } containerType
+                    && metadata.PropertyName is { } propertyName)
+                {
+                    var propertyDocId = DocumentationCommentIdHelper.CreateDocumentationId(containerType, propertyName);
+                    if (XmlCommentCache.Cache.TryGetValue(DocumentationCommentIdHelper.NormalizeDocId(propertyDocId), out var propertyComment))
+                    {
+                        var parameter = operation.Parameters?.SingleOrDefault(p => p.Name == metadata.Name);
+                        if (parameter is null)
+                        {
+                            if (operation.RequestBody is not null)
+                            {
+                                operation.RequestBody.Description = propertyComment.Summary ?? propertyComment.Description;
```

Comment:
Is this the right precedence? Just from the naming it looks like it should be the other way around.

Reply:
I believe you're right. I believe this should be .Value, .Returns then .Summary to match other property-based comments.